app/src/main/java/com/safetyheads/akademiaandroida/CardViewSectionsFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.safetyheads.akademiaandroida
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+class CardViewSectionsFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
```

Comment:
I think we can delete this fragment since you are using this card in task number 22